docker/docker-compose.yml

```diff
@@ -33,21 +33,21 @@ services:
     depends_on:
       - clamd
 
-localstack:
-  image: localstack/localstack:latest
-  ports:
-    - 9000:8080
-    - 4572:4572
-    - 4576:4576
-    - 4575:4575
-    - 4571:4571
-    - 4578:4578
-  networks:
-    - hocs-network
-  environment:
-    HOSTNAME_EXTERNAL: localstack
-    DEFAULT_REGION: eu-west-2
-    SERVICES: sqs,s3,sns
+  localstack:
+    image: localstack/localstack:latest
```

Comment:
I highly doubt this even spins up correctly anymore. Since 0.11.6 all services run off an edge server: https://github.com/localstack/localstack/releases/tag/v0.11.6